lib/rspec/core/configuration.rb

```diff
@@ -2034,7 +2034,8 @@ def load_file_handling_errors(method, file)
         __send__(method, file)
       rescue Support::AllExceptionsExceptOnesWeMustNotRescue => ex
         relative_file = Metadata.relative_path(file)
-        reporter.notify_non_example_exception(ex, "An error occurred while loading #{relative_file}.")
+        suggestions = (ex.is_a? LoadError) ? DidYouMean.new(relative_file).call : nil
+        reporter.notify_non_example_exception(ex, "An error occurred while loading #{relative_file}.#{suggestions}")
```

Comment:
I wonder if here we should do:
```
rescue LoadError => ex
  relative_file = Metadata.relative_path(file)
  reporter.notify_non_example_exception(ex, "An error occurred while loading #{relative_file}.#{DidYouMean.new(relative_file).call}")
rescue Support::AllExceptionsExceptOnesWeMustNotRescue => ex
```

Reply:
I like that.

Reply:
done. Note that I added in
`RSpec.world.wants_to_quit = true`, to the LoadError rescue block, I am not sure if it is needed.

Reply:
Ah yes, it is.